core/controllers/learner_playlist.py

```diff
@@ -27,12 +27,31 @@ class LearnerPlaylistHandler(base.BaseHandler):
     @acl_decorators.can_access_learner_dashboard
     def post(self, activity_type, activity_id):
         position_to_be_inserted_in = self.payload.get('index')
+
+        playlist_limit_exceeded = False
+        belongs_to_completed_or_incomplete_list = False
```

Comment:
Let's say we always prioritize error message (or informational text) A over error message B. If A occurs, does it make any difference to the frontend whether B occurs?

If not, then really the piece of info to pass to the frontend is just the single prioritized message.